### PR TITLE
751 Add authToken to queries send through RTK

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/rtk/api.ts [hunk 1]
```diff
@@ -26,11 +38,17 @@ export const orchestratorApi = createApi({
             case BaseQueryTypes.fetch:
                 const fetchFn = fetchBaseQuery({
                     baseUrl: orchestratorApiBaseUrl,
+                    prepareHeaders: (headers) => {
```
Comment (tjeerddie) [inline packages/orchestrator-ui-components/src/rtk/api.ts:41]: I don't think the function is necessary here and you can just do `prepareHeaders: prepareHeaders` instead.

**Review by tjeerddie — APPROVED** [review]
LGTM

File: packages/orchestrator-ui-components/src/rtk/api.ts (hunk 1 above)
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/rtk/api.ts:41]: think so too and if key value have the same name `prepareHeaders,` is sufficient

File: packages/orchestrator-ui-components/src/rtk/api.ts [hunk 2]
```diff
@@ -13,6 +17,14 @@ type ExtraOptions = {
     baseQueryType?: BaseQueryTypes;
 };
 
+const prepareHeaders = async (headers: Headers) => {
+    const session = (await getSession()) as SessionWithToken;
+    if (session && session.accessToken) {
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/rtk/api.ts:22]: this could be session?.accessToken

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/rtk/api.ts (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/rtk/api.ts:22]: Done

File: packages/orchestrator-ui-components/src/rtk/api.ts (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/rtk/api.ts:41]: Done

**Pull request merged**